### DISCUSSION TITLE
Where is located the style for Vanilla CSS used in Starter kits?

### DISCUSSION THREAD

**Discussion by artola** [adobe/react-spectrum#6162] [Q&A] [answered]
I can download a ZIP file or Preview, but I am looking for the source of the CSS. is it located in other repo or non-public one?

![image](https://github.com/adobe/react-spectrum/assets/11500763/545e5af6-1ebb-4ad8-88b6-fd54709f6d94)

https://react-spectrum.adobe.com/react-aria/Button.html

**Comment by snowystinger** [comment, marked as answer]
I think this is what you're looking for
https://github.com/adobe/react-spectrum/blob/e6923d9473264b972974513dacbcfab5a24d5616/scripts/extractStarter.mjs#L99
which shows us the theme is defined here https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/example-theme/src/index.css

and then the actual CSS if we open the starter shows and import from that theme, plus some styles and selectors
Downloads/react-aria-starter.48e54b651/src/Button.css
This CSS matches the expanded CSS under the examples on the docs page you linked to. https://react-spectrum.adobe.com/react-aria/Button.html

We pull the CSS out of our docs and place it in the starter example.

**Reply by artola** [reply]
@snowystinger Thanks a lot! We are using `react-aria-components` as a _blueprint_ and it is very handy to detect changes.
